Docs/source/install/dependencies.rst

```diff
@@ -74,6 +74,14 @@ Brew (macOS/Linux)
    brew install pkg-config  # for fftw
    brew install open-mpi
 
+.. note::
+
+   On macOS, it is sometimes required to also set the following environment variable to hint the MPI installation location inside ``brew``'s prefix:
+
+   .. code-block:: bash
+
+      export MPI_HOME=/usr/local
```

Comment:
Oh wait, I confused this now. Setting `MPI_HOME` did not help in that situation...